win32/src/winapi/bitmap.rs

```diff
@@ -16,6 +16,23 @@ pub enum BI {
     PNG = 5,
 }
 
+#[repr(C)]
+#[derive(Debug)]
+pub struct BITMAPCOREHEADER {
+    pub bcSize: DWORD,
+    pub bcWidth: WORD,
+    pub bcHeight: WORD,
+    pub bcPlanes: WORD,
+    pub bcBitCount: WORD,
+}
+unsafe impl memory::Pod for BITMAPCOREHEADER {}
+impl BITMAPCOREHEADER {
+    pub fn stride(&self) -> u32 {
```

Comment:
It looks like this type is a strict subset of BITMAPINFOHEADER.  Would it make sense to just make one of these the first member of BITMAPINFOHEADER or something like that?  Feels wrong to duplicate this.

Reply:
`BITMAPCOREHEADER`'s `bcWidth` and `bcHeight` are 16-bit, but `BITMAPINFOHEADER`'s are 32-bit.

Reply:
👍